src/components/PeopleManagement/LearnerDetailPage/LearnerDetailPage.jsx

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { logError } from '@edx/frontend-platform/logging';
+import {
+  Breadcrumb, Card, Icon, Skeleton,
+} from '@openedx/paragon';
+import { Person } from '@openedx/paragon/icons';
+
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import LmsApiService from '../../../data/services/LmsApiService';
+import { ROUTE_NAMES } from '../../EnterpriseApp/data/constants';
+import formatDates from '../utils';
+import { useEnterpriseGroupUuid } from '../data/hooks';
+
+const LearnerDetailPage = ({ enterpriseUUID }) => {
+  const { enterpriseSlug, groupUuid, learnerId } = useParams();
+  const { data: enterpriseGroup } = useEnterpriseGroupUuid(groupUuid);
```

Comment:
[opportunity] Could we typescript annotate the [useEnterpriseGroupUuid](https://github.com/openedx/frontend-app-admin-portal/blob/master/src/components/PeopleManagement/data/hooks/useEnterpriseGroupUuid.js) hook?